vuepress/docs/next/tutorials/cms/page-management.md

```diff
@@ -0,0 +1,130 @@
+---
+redirectFrom: /next/tutorials/cms/publish-a-content-tutorial.html
+---
+
+# Page Management
+
+The App Builder provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to layout the available frames on a page.
+
+## Create a Page
+
+- Go to `Pages → Management`
+- Click **Add**
+
+![image](./extracted-media/media/Publish1.png)
+
+A page must have a title, a code, a placement in the page tree, an owner
+group, a join group and a page template. When you create a new page, you
+must:
+
+-   Provide a title and a code in the Info section.
+
+-   Provide, selecting them from the dropdown menus, an Owner Group and, optionally,
+    a Join Group in the Page Groups section,
+
+-   Select a Page Template from the available templates.
+
+![image](./extracted-media/media/Publish2.png)
+
+-   Click the green **Save and Configure** button. This brings you to
+    the configuration section, which lists all the frames defined in the
+    page template you associated with that page.
+
+![image](./extracted-media/media/Publish3.png)
+
+-   Add widgets by dragging them from the right column on to the desired frame. 
+-   If you select the **Content** widget, its corresponding settings page will be shown.
+
+![image](./extracted-media/media/Publish4.png)
+
+-   Click **Add existing content** to bring up the Content selection view.
```

Comment:
Looking at the screenshot on this one I wonder if we should explain the colors in the status column for the content. Or grab a new screenshot with all yellow content. The yellow dot stands out.

Reply:
Good catch.